DataLoader/test/data-loader.test.js

```diff
@@ -0,0 +1,105 @@
+let Leaflet = require('../../verify/leaflet.js')
+let { PlantquestDataLoader } = require('../dist/etage-control.umd.cjs')
+
+describe('DataLoader', () => {
+  const options = {
```

Comment:
these are the options for Etage I think?

Reply:
Indeed. Not playing with tests yet.